stage6/09-sense-emu/00-run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+pip3 install sense-emu
```

Comment:
Does this pip package contains everything that was installed by apt (tools, docs etc) ?

Reply:
Yes this package contains the tools (sense_csv, sense_emu_gui, sense_play and sense_rec) and the python3 module. The docs I presumed is also included since one can open the examples scripts and the documentations from the GUI. To build the docs, it is advised to clone the [repo](https://github.com/astro-pi/python-sense-emu) and build manually